docs/WebView/README.en.md

```diff
@@ -122,6 +122,27 @@ When performing an iOS build in certain Unity versions, the buttons on **Navigat
 
 ![GPMWebViewBundle.png](images/GPMWebViewBundle.png)
 
+#### Optional automation
+
+You can automate above process by using [OnPostprocessBuild](https://docs.unity3d.com/ScriptReference/Build.IPostprocessBuildWithReport.OnPostprocessBuild.html) and [PBXProject](https://docs.unity3d.com/ScriptReference/iOS.Xcode.PBXProject.html).
+
+Example, inside `OnPostprocessBuild()` add:
+
+Setting Other Linker Flags
+
+```cs
+// adding -ObjC to Other Linker Flags in Build Settings
+pbxProject.AddBuildProperty(appGuid, "OTHER_LDFLAGS", "-ObjC");
+```
+ 
+GPMWebView.bundle
+
+```cs
+// adding GPMWebView.bundle to Copy Bundle Resources in Build Phases
+var webViewBundleGuid = pbxProject.AddFile("Frameworks/GPM/WebView/Plugins/IOS/GPMWebView.bundle", "GPMWebView.bundle", PBXSourceTree.Build);  
+pbxProject.AddFileToBuild(appGuid, webViewBundleGuid);
```

Comment:
When '-ObjC' or 'GPMWebView.bundle' is present, it should be checked to ensure that it works without any problems, or a supplementary description of the content should be provided.

Reply:
OK, I will add checking code and test around this then update.

Reply:
I have checked! They are already checked by Unity and Xcode!

[AddBuildProperty](https://docs.unity3d.com/ScriptReference/iOS.Xcode.PBXProject.AddBuildProperty.html) ignores duplicate values. So only one -ObjC is added!

Same for GPMWebView.bundle, it's not possible to add again programmatically and manually.

To make sure, I have tested with multiple lines:

```cs
                // Add linker
                pbxProject.AddBuildProperty(appGuid, "OTHER_LDFLAGS", "-ObjC");

                pbxProject.AddBuildProperty(appGuid, "OTHER_LDFLAGS", "-ObjC");

                pbxProject.AddBuildProperty(appGuid, "OTHER_LDFLAGS", "-ObjC");

                pbxProject.AddBuildProperty(appGuid, "OTHER_LDFLAGS", "-ObjC");

                // Add bundle
                var webViewBundleGuid = pbxProject.AddFile("Frameworks/GPM/WebView/Plugins/IOS/GPMWebView.bundle", "GPMWebView.bundle", PBXSourceTree.Build);  
                pbxProject.AddFileToBuild(appGuid, webViewBundleGuid);

                pbxProject.AddFileToBuild(appGuid, webViewBundleGuid);

                pbxProject.AddFileToBuild(appGuid, webViewBundleGuid);
```
Only one'-ObjC' and 'GPMWebView.bundle' were added. There is no error or problems.

I have provided additional description for this. 👍